ui/css/design-system/attributes.scss

```diff
@@ -38,11 +38,11 @@ $fractions: (
 
 $sizes-numeric: 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12;
 
-$sizes-strings: xs, sm, md, lg, xl, none;
+$sizes-strings: xs, sm, md, lg, xl;
 
 $border-style: solid, double, none, dashed, dotted;
 $directions: top, right, bottom, left;
-$display: block, grid, flex, inline-block, inline-grid, inline-flex, list-item;
+$display: block, flex, grid, inline-block, inline-grid, inline-flex, list-item, none;
```

Comment:
Not really sure if this matters but by rearranging these, `grid` will supersede `flex` if there is somehow a specificity issue just with the way sass is looping over each style